plugins/hypervisors/xenserver/src/com/cloud/hypervisor/xenserver/resource/Xenserver625StorageProcessor.java

```diff
@@ -538,7 +562,7 @@ public Answer backupSnapshot(final CopyCommand cmd) {
                     final String[] tmp = result.split("#");
                     snapshotBackupUuid = tmp[0];
                     physicalSize = Long.parseLong(tmp[1]);
-                    finalPath = folder + File.separator + snapshotBackupUuid;
+                    finalPath = folder + File.separator + snapshotBackupUuid + ".vhd";
```

Comment:
While I would want to add the extension, but I think we should revert this specific change, as otherwise it would add upgrade complexities. Given the short time for next RCs, such a change can be properly addressed in future with tooling around upgrading/renaming the files and paths.